Documentation/data-cleanup/python-script/program_deletion.py

```
@@ -0,0 +1,157 @@
+        access_token = get_access_token()
+        all_program_ids = fetch_all_program_ids(access_token)
+
+        remaining_programs = delete_required_program_ids(
+            all_program_ids,
+            list_of_required_program_ids
+        )
+        print(remaining_programs)
+        logger.info("Final program count after deletion: " + str(len(remaining_programs)))
+        logger.info("Script executed successfully.")
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)